01-getting-started/installation/platforms/install-on-gcp.md

```diff
@@ -0,0 +1,195 @@
+---
+Description: >-
+  Learn how to quickly deploy Squidex (with MongoDB) on Google Cloud Platform using Cloud Run & MongoDB Atlas.
+---
+
+# Deploy Squidex on Google Cloud Platform using Cloud Run, Cloud Storage & MongoDB Atlas
+
+This version of the installation tutorial deploys Squidex as a container on [Cloud Run](https://cloud.google.com/run/) in Google Cloud Platform. The approach is PaaS or Managed service.
+
+The tutorial does not cover the basics of GCP. You should be familiar with them before you begin with the installation instructions. It uses the following services from GCP:
+* [Google Cloud Storage](https://cloud.google.com/storage)
+* [Google Cloud Run](https://cloud.google.com/run/)
+* [MongoDB Atlas](https://www.mongodb.com/cloud/atlas/mongodb-google-cloud) running on Google Cloud
+
+This tutorial runs MongoDB in a free tier mode is recommended for non-production or trial environments. For production environments it is recommended to go for the Pay-as-you-go model available at the Google Cloud [marketplace](https://console.cloud.google.com/marketplace/product/mongodb/mdb-atlas-self-service).
+
+Use this tutorial as a getting started guide in GCP to explore Squidex or use it as a Dev/Sandbox environment. This setup may not be recommended for a production environment. 
```

Comment:
I think we should make it very clear why it is not recommended.

Reply:
Added details.